lib/migration.js

```diff
@@ -111,7 +115,7 @@ function mixinMigration(MySQL, mysql) {
     });
   };
 
-  MySQL.prototype.alterTable = function(model, actualFields, actualIndexes, done, checkOnly) {
+  MySQL.prototype.alterTable = function(model, actualFields, actualIndexes, actualFks, done, checkOnly) {
```

Comment:
What's the best/preferred way to do that? Should it just do typeof checks for the last 3 parameters to see which is which?